Source/ACE/Network/GameEvent/Events/GameEventIdentifyObjectResponse.cs

```diff
@@ -282,5 +285,177 @@ private string DebugOutputString(System.Type type, WorldObject obj)
 
             return debugOutput;
         }
+
+        private void WriteIdentifyObjectHeader(BinaryWriter writer, IdentifyResponseFlags flags, bool success)
```

Comment:
These functions could be private static void -- they don't use any object local data -- they are purely functional.